app/src/test/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdateTest.kt

```diff
@@ -207,4 +210,24 @@ class CustomDrugEntryUpdateTest {
             )
         )
   }
+
+  @Test
+  fun `when drug frequency choice items are loaded, then update the model`() {
+    val listItems = listOf(
+        DrugFrequencyChoiceItem(drugFrequency = null, labelResId = R.string.custom_drug_entry_sheet_frequency_none),
+        DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.OD, labelResId = R.string.custom_drug_entry_sheet_frequency_OD),
+        DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.BD, labelResId = R.string.custom_drug_entry_sheet_frequency_BD),
+        DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.QDS, labelResId = R.string.custom_drug_entry_sheet_frequency_QDS),
+        DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.TDS, labelResId = R.string.custom_drug_entry_sheet_frequency_TDS))
```

Comment:
Will resolve this in another PR

Reply:
I'm planning to use `DrugFrequencyChoiceItems(drugFrequencyFactory.provideFields()).items` instead of defining the list items myself now. This will help avoid the usage of string resources in the test suite. Does this sound like a good approach? @vinaysshenoy